staging/src/k8s.io/kubectl/pkg/util/prune/prune_test.go

```diff
@@ -48,23 +48,33 @@ func (m *testRESTMapper) RESTMapping(gk schema.GroupKind, versions ...string) (*
 
 func TestGetRESTMappings(t *testing.T) {
```

Comment:
This seems to be missing coverage for the case where an allow list was explicitly provided. Can we add one, or am I missing that that's covered elsewhere? Ideally I think we'd also have tests for manual lists + namespace provided or not, noting that once we change the behaviour for the default list, it should NOT affect the behaviour for custom lists.

Reply:
I added one. 
But from code-coverage aspect, it does not cover more as the default resource is not empty.

```
ok      k8s.io/kubernetes/vendor/k8s.io/kubectl/pkg/util/prune  0.475s  coverage: 84.4% of statements
```